lib/galaxy/model/__init__.py

```diff
@@ -2035,7 +2035,7 @@ def get_size(self, nice_size=False):
         return self.dataset.get_size()
 
     def set_size(self):
-        """Returns the size of the data on disk"""
+        """Sets the size of the data on disk"""
```

Comment:
This method actually sets *and* returns the dataset size.

Reply:
Thanks, updated.